src/main/resources/templates/mail/notification/vcsAccessTokenAddedEmail.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+
+<th:block th:replace="mail/notification/fragments :: head"/>
+
+<body>
+<!-- Message Body Start -->
+<th:block th:replace="mail/notification/fragments :: header" />
+
+<div id="message-body">
+    <th:block th:replace="mail/notification/fragments :: greetingsFullName" />
+
+    <p>
+    <span th:text="#{email.notification.vcsAccessTokenAdded.title}">The following SSH key was added to your account:</span>
+    </p>
+
+    <p>
+        <span th:text="#{email.notification.vcsAccessTokenAdded.ifMistake}">If adding it was by mistake, remove it here:</span>
```

Comment:
```suggestion
        <span th:text="#{email.notification.vcsAccessTokenAdded.ifMistake}">If adding it was by mistake, you can remove it here:</span>
```

Reply:
Should we also mention something like: "If you did not add the ssh key, please report this to your administrator" and provide the Artemis instance email contact?

Reply:
I added an additional paragraph, with the Artemis instance's email contact